src/stepFunctions/wizards/publishStateMachineWizard.ts

```diff
@@ -81,6 +81,9 @@ export class DefaultPublishStateMachineWizardContext extends WizardContext imple
     private readonly iamClient: IamClient
     private readonly stepFunctionsClient: StepFunctionsClient
 
+    private readonly totalSteps = 2
```

Comment:
Can this not be calculated instead of hardcoded in each wizard? Does `WizardContext` not have something like `addStep()` or a list of steps?